src/components/TextAreaChat/index.tsx

```diff
@@ -85,7 +85,7 @@ const TextAreaChat = ({
         placeholderTextColor={COLORS.gray400}
         style={[
           S.textArea,
-          {color: disabledInput ? COLORS.lightgrey : COLORS.white2, minHeight, maxHeight}
+          {color: disabledInput ? COLORS.gray100 : COLORS.white, minHeight, maxHeight}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color change from `COLORS.lightgrey` to `COLORS.gray100` and `COLORS.white2` to `COLORS.white` seems fine as it's a part of the task to update colors across various components. However, ensure that these new color constants are defined and imported correctly from the `utils/theme` module. Also, verify that this color change doesn't affect the readability and accessibility of the text area in different themes (light/dark mode).
```diff
- {color: disabledInput ? COLORS.lightgrey : COLORS.white2, minHeight, maxHeight}
+ {color: disabledInput ? COLORS.gray100 : COLORS.white, minHeight, maxHeight}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->